encord/project.py

```diff
@@ -117,21 +118,38 @@ def ontology(self) -> Dict[str, Any]:
         This method returns the same structure as :meth:`encord.Project.ontology_structure`, just in
         raw python dictionary format.
         """
-        return self._ontology.structure.to_dict()
+        return self._ontology_instance.structure.to_dict()
```

Comment:
In a non-ideal scenario when the customer relies on `_ontology`, they'll get exception when trying to call methods from `None` object. I guess we should expose `_ontology` property and rename the private attribute to something like `__lazy_load_ontology` so at least it wouldn't break crazy use cases

Reply:
This is a fair point but I think that we can probably change the private internals without it being too much of an issue. Have we checked that all of our private code doesn't use `_ontology` directly? Need to ensure that we don't attempt to actually use it in label parsing for example and we haven't initialised it?

E.G: Can we ensure that we have always fetched the Ontology before we attempt to parse label blobs?

Reply:
I think it's totally fair to break backwards compat in underscore-prefix parts.

Reply:
+1, that's not a part of the public interface, so no problem here.
And this method is deprecated for more than a year anyways.